meinberlin/apps/documents/templates/meinberlin_documents/react_documents.html

```diff
@@ -1,5 +1,7 @@
 {% load static %}
 
+  <link href="/static/django_ckeditor_5/dist/styles.css" type="text/css" media="all" rel="stylesheet">
+
 {{ ckeditor_media.js }}
```

Comment:
this was here already but means that there will be a script tag just randomly in the body, should we move this somewhere else?